taxi/forms.py

```diff
@@ -0,0 +1,42 @@
+from django import forms
+from django.core.exceptions import ValidationError
+from django.contrib.auth.forms import UserCreationForm
+
+from .models import Driver, Car
+
+
+def validate_license_number(value):
+    if len(value) != 8:
+        raise ValidationError("License number must have 8 characters!")
+    elif not value[0:3].isalpha():
```

Comment:
there's no point in using `elif` after `raise`.
Also, slices can be shortened to e.g. just [:3].
These apply to all such cases in this review